src/components/DataTablePageResults/DataTablePageResults.tsx

```diff
@@ -24,7 +24,7 @@ const DataTablePageResults = ({
   const page = offset / limit;
   const startTotal = () => (page * limit + 1)
   return (
-    <p className={`ds-u-margin-bottom--2 ${className}`}>
```

Comment:
This was moved to the ResourceHeader component to preserve this style on the /data route